snowplow.js

```diff
@@ -41,8 +43,12 @@ const setupBrowserTracker = () => {
     }
   }
 
-  newTracker('snplow5', 'https://collector.snowplow.io', trackerConfig)
-  newTracker('biz1', 'https://c.snowplow.io', trackerConfig)
+  return trackerConfig;
+};
+
+const setupBrowserTracker = () => {
+  newTracker('snplow5', 'https://collector.snowplow.io', createTrackerConfig('_sp5_'))
+  newTracker('biz1', 'https://c.snowplow.io', createTrackerConfig('_sp_biz1_'))
```

Comment:
I think we should add the new cookie here: https://github.com/snowplow/documentation/blob/main/src/pages/cookie-preferences.mdx?plain=1#L64

Reply:
Good point, done!